x-pack/legacy/plugins/canvas/public/components/toolbar/__examples__/toolbar.stories.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/*
+ TODO: uncomment and fix this test to address storybook errors as a result of nested component dependencies
+ */
```

Comment:
@maggieghamry sorry to be a pain! But could you create that issue we talked about and then just copy the link for it into this TODO message so we know where that issue lives and don't have to search for it?

Reply:
@poffdeluxe sorry, I created the issue and linked it in here, not in the code. I'll add it to the code as well